src/lib/workspace/AllAnalyses.tsx

```diff
@@ -0,0 +1,338 @@
+            return newSet;
+          }),
+        onRowDeselect: (analysis: Analysis) =>
+          setSelectedAnalyses((set) => {
+            const newSet = new Set(set);
+            newSet.delete(analysis.id);
+            return newSet;
+          }),
+        onMultipleRowSelect: (analyses: Analysis[]) =>
+          setSelectedAnalyses((set) => {
+            const newSet = new Set(set);
+            for (const analysis of analyses) newSet.add(analysis.id);
+            return newSet;
+          }),
+        onMultipleRowDeselect: (analyses: Analysis[]) =>
+          setSelectedAnalyses((set) => {
+            const newSet = new Set(set);
+            for (const analysis of analyses) newSet.delete(analysis.id);
+            return newSet;
+          }),
+      },
+      uiState: {
+        sort: {
+          columnKey: tableSort[0],
+          direction: tableSort[1],
+        },
+      },
+      columns: [
+        {
+          key: 'name',
+          name: 'Analysis',
+          sortable: true,
+          renderCell: (data: { row: Analysis }) => (
+            <>
+              <Tooltip
+                title={
+                  isPinnedAnalysis(data.row.id)
+                    ? 'Remove from pinned analyses'
+                    : 'Add to pinned analyses'
+                }
+              >
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      size="small"
+                      icon={
+                        <Icon
+                          style={{ color: '#aaa' }}
+                          className="fa fa-thumb-tack"
+                        />
+                      }
+                      checkedIcon={
+                        <Icon color="primary" className="fa fa-thumb-tack" />
+                      }
+                      checked={isPinnedAnalysis(data.row.id)}
+                      onChange={(e) => {
+                        if (e.target.checked) addPinnedAnalysis(data.row.id);
+                        else removePinnedAnalysis(data.row.id);
+                      }}
+                    />
+                  }
+                  label=""
+                />
+              </Tooltip>
+              <Link
+                to={Path.join(
+                  history.location.pathname,
+                  data.row.studyId,
+                  data.row.id
+                )}
+              >
+                {data.row.name}
+              </Link>
+            </>
+          ),
+        },
+        {
+          key: 'study',
+          name: 'Study',
+          sortable: true,
+          renderCell: (data: { row: Analysis }) => {
+            const dataset = datasets?.records.find(
+              (d) => d.id[0].value === data.row.studyId
+            );
+            if (dataset == null) return 'Unknown study';
+            return (
+              <Link to={`${url}/${dataset.id[0].value}`}>
+                {dataset.displayName}
```

Comment:
SCORE S. mansoni italics tags are currently showing.  The approach in StudyList can be used here to clean the HTML.

```
import { safeHtml } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
```

```suggestion
                {safeHtml(dataset.displayName)}
```

Reply:
Thanks, this is good! I will update soon.